lib/src/Uncertainty/Algorithm/MetaModel/Kriging/KrigingResult.cxx

```diff
@@ -181,7 +177,7 @@ void KrigingResult::computeF() const
   // Nothing to do if the design matrix has already been computed
   if (F_.getNbRows() != 0) return;
   const UnsignedInteger outputDimension = covarianceModel_.getOutputDimension();
-  const UnsignedInteger sampleSize = inputSample_.getSize();
+  const UnsignedInteger sampleSize = getInputSample().getSize();
```

Comment:
The  MetaModelResult parent class (at least through the constructor that is used in this case) has an inputSample_ attribute, isn't it easier to keep the code here as it is?

Reply:
yes it has but the attribut is private. Thus no choice....

Reply:
oh, I see